tests/IntegrationTest.php

```diff
@@ -261,6 +261,12 @@ public function provideRenderStimulusController()
             'controllerValues' => ['myValue' => 'scalar-value'],
             'expected' => 'data-controller="my-controller" data-my-controller-my-value-value="scalar-value"',
         ];
+
+        yield 'false-attribute-value-is-not-rendered' => [
+            'dataOrControllerName' => 'false-controller',
+            'controllerValues' => ['isEnabled' => false],
+            'expected' => 'data-controller="false-controller"',
+        ];
```

Comment:
I did some more digging into this... and I think my proposed solution (which you implemented perfectly) is not... perfect ;).

For this test case:

```html
<!-- current output -->
<div
    data-controller="false-controller"
    data-false-controller-is-enabled-value=""
>
```

The `=""` for a `false` value is the problem. It should be `="false"`, which I've determined DOES cause the value in the controller to be `false`. Similarly, for `true`, the attribute should be `="true"`. Additionally, omitting the attribute entirely would cause `this.hasIsEnabledValue` (you can use a "hasser" like this to check for the existence of a value) to incorrectly return false with this current PR.

So I think the fix is to "fix" how we convert boolean values:

```
<!-- if isEnabled is passed as FALSE -->
<div
    data-controller="false-controller"
    data-false-controller-is-enabled-value="false"
>

<!-- if isEnabled is passed as TRUE -->
<div
    data-controller="false-controller"
    data-false-controller-is-enabled-value="true"
>
```

Reply:
that make sense.. done.